.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,21 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: false
```

Comment:
@felixfontein do the announcement authors know to manually go to https://github.com/ansible-collections/news-for-maintainers/issues/new?